includes/command.inc

```diff
@@ -523,11 +523,22 @@ function drush_handle_command_output($command, $structured_output) {
  */
 function _drush_verify_cli_options($command) {
 
-  // Start out with just the options in the current command record.
-  $options = _drush_get_command_options($command);
+  if (annotation_adapter_is_annotated($command)) {
+    // The canonical options list is now in $application
+    $application = Drush::getApplication();
+    annotation_adapter_add_legacy_commands_to_application($application);
```

Comment:
Might we end up doing this multiple times, or is _drush_verify_cli_options() guaranteed to be called only once (including redispatches, etc)? If called multiple times, would that cause any problems?